plugins/org.eclipse.n4js.model/model/ProjectDescription.xcore

```diff
@@ -106,14 +107,16 @@ class BootstrapModule {
 }
 
 enum ProjectType {
-	VALIDATION = 0,
-	APPLICATION = 1,
-	PROCESSOR = 2,
-	LIBRARY = 3,
-	API = 4,
-	RUNTIME_ENVIRONMENT = 5,
-	RUNTIME_LIBRARY = 6,
-	TEST = 7
+	//PLAINJS = 0,
```

Comment:
Is this safe with regard to https://github.com/eclipse/n4js/blob/master/plugins/org.eclipse.n4js/src/org/eclipse/n4js/packagejson/PackageJsonHelper.java#L277 ?